cdap-app-fabric/src/main/java/co/cask/cdap/metadata/MetadataHttpHandler.java

```diff
@@ -840,7 +840,8 @@ public void searchMetadata(HttpRequest request, HttpResponder responder,
                              // 2147483647 is Integer.MAX_VALUE
                              @QueryParam("limit") @DefaultValue("2147483647") int limit,
                              @QueryParam("numCursors") @DefaultValue("0") int numCursors,
-                             @QueryParam("cursor") @DefaultValue("") String cursor) throws Exception {
+                             @QueryParam("cursor") @DefaultValue("") String cursor,
+                             @QueryParam("showHidden") boolean showHidden) throws Exception {
```

Comment:
It wasn't clear to me that this would default to false (since its a query param, and didn't have the DefaultValue annotation). I think its better to be explicit in this case.
See the parameter numCursors just 2 lines above.
It's explicit, even though the default value is 0. This inconsistency can be confusing.